docs/encoder.md

```diff
@@ -0,0 +1,48 @@
+# Encoder
+Add twist control to your keyboard! Volume, zoom, anything you want.
+
+## Enabling the extension
+The constructor takes a minimun of 3 arguments: a list of pad_a pins, a list of pad_b pins, 
+and an encoder_map.  The encoder_map is modeled after the keymap and works the
+same way. It shoul have as many layers as your keymap, and use filler keys for 
+layers that you don't require any action.  The encoder supports a velocity mode
+if you desire to make something for video or sound editing. The direction of 
+increment/decrement can be changed to make sense for the direction the knob is 
+turning by setting the is_inverted flag.
+
+## Configuration
+Create your special keys:
+```

```

Comment:
Any python code blocks should start with 
\`\`\`python

in order to get syntax highlighting.

Reply:
Added to all blocks of python codee